tasks/pre_checks.yml

```diff
@@ -71,22 +71,25 @@
     - not k3s_skip_validation
     - not k3s_skip_env_checks
 
-- include_tasks: pre_checks_version.yml
+- name: Ensure version pre-checks
```

Comment:
@xanmanning - The task name here and with some others begins with `ensure`, but the file begins with `pre_checks`.

Do you have a preference/feedback on whether this should be changed to something else?

Reply:
Maybe `Run version pre-checks` and `Run cgroups pre-checks` and so on ?